woocommerce/woocommerce
# Marketplace Themes: Feature Branch

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes 18221-gh-Automattic/woocommerce.com.

This PR contains the changes from:

|PR|Description|
|---|---|
|https://github.com/woocommerce/woocommerce/pull/40247|Adds a themes tab to the marketplace. Closes 18032-gh-Automattic/woocommerce.com.|
|https://github.com/woocommerce/woocommerce/pull/40272|Adds horizontal scrolling when tabs overflow.|
|https://github.com/woocommerce/woocommerce/pull/40302|Adds theme cards.|
|https://github.com/woocommerce/woocommerce/pull/40303|Moves search request into `Content` and adds `SearchResults` component.|
|https://github.com/woocommerce/woocommerce/pull/40333|Adds support for selecting categories.|
|https://github.com/woocommerce/woocommerce/pull/40368|Updates labels and icon in the NoResults component.|
|https://github.com/woocommerce/woocommerce/pull/40375|Adds "View all" button to search results. Closes 18213-gh-Automattic/woocommerce.com.|
|https://github.com/woocommerce/woocommerce/pull/40389|Adds support for themes group on the Discover tab.|



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Please check out this branch and run a build.
2. Browse through the [Extensions](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions) marketplace. Go to the Themes tab and see themes displayed.
3. Do a search, and note that you get results for themes as well as extensions.
4. Notice what you get when you do a search with no results for either or both of these product types.
5. Check that the Discover tab still works.
6. Please try different viewport sizes to make sure the responsive layouts are OK.

For more detailed instructions, please see the individual PRs.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add Themes to the Extensions catalogue for easy download and installation.

#### Comment 

</details>
